File: add/codesnippet/CPP/m-system.runtime.remotin_68_1.cpp
   HttpChannel^ myClientChannel = gcnew HttpChannel( myProperties,gcnew SoapClientFormatterSinkProvider,gcnew SoapServerFormatterSinkProvider );
   ChannelServices::RegisterChannel( myClientChannel, false );
   
   // Get the registered channel. 
   Console::WriteLine( "Channel Name : {0}", ChannelServices::GetChannel( myClientChannel->ChannelName )->ChannelName );
   Console::WriteLine( "Channel Priorty : {0}", ChannelServices::GetChannel( myClientChannel->ChannelName )->ChannelPriority );
   

File: add/codesnippet/CSharp/m-system.runtime.remotin_68_1.cs
         HttpChannel myClientChannel = new HttpChannel(myProperties,
            new SoapClientFormatterSinkProvider(),
            new SoapServerFormatterSinkProvider());
         ChannelServices.RegisterChannel(myClientChannel);
         // Get the registered channel. 
         Console.WriteLine("Channel Name : "+ChannelServices.GetChannel(
            myClientChannel.ChannelName).ChannelName);
         Console.WriteLine("Channel Priorty : "+ChannelServices.GetChannel(
            myClientChannel.ChannelName).ChannelPriority);